src/runtime.js

```diff
@@ -119,6 +120,7 @@ export function installRuntimeServices(global) {
   installTimerService(global);
   installVsyncService(global);
   installXhrService(global);
+  installCachedXhrService(global);
```

Comment:
Certainly, here are the results.

| |max|min|gzip|
|--|--|--|--|
|**master**|1.11 MB|245.71 kB|amp.js
|**PR #7562**|1.11 MB|246.56 kB|amp.js